### ISSUE TITLE
[jx import] Makefile missing when using `jx import`

### ISSUE THREAD

**Issue by cyrille-leclerc** [jenkins-x/jx#184]
`jx import`does not create the Makefile `charts/<MY_APP>/Makefile`.

See https://github.com/cloudbeers/draft-spring-boot-app/commit/90e95e2a1ab9a3f441de08f7aa46027026af45d0

See https://gist.github.com/cyrille-leclerc/f82afdd129b4516e6f84e0e7a4e5a860

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 37.388 s
[INFO] Finished at: 2018-02-01T09:20:19+00:00
[INFO] Final Memory: 17M/25M
[INFO] ------------------------------------------------------------------------
[Pipeline] }
[Pipeline] // container
[Pipeline] dir
Running in /home/jenkins/workspace/raft-spring-boot-app_master-UYE7LYZTRFK2QPXLTMX2XRNZQF2OAOQU5YJA6IJXVYDFB3ZANCRQ/charts/draft-spring-boot-app
[Pipeline] {
[Pipeline] container
[Pipeline] {
[Pipeline] sh
[draft-spring-boot-app] Running shell script
+ make tag
make: *** No rule to make target `tag'.  Stop.
[Pipeline] }
[Pipeline] // container
[Pipeline] }
[Pipeline] // dir
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Deploy Staging)
Stage 'Deploy Staging' skipped due to earlier failure(s)
```

**Comment by rawlingsj** [comment]
Can you double check the steps to recreate the problem please?  I just tried to recreate:
-  I went to http://start.spring.io/
- added web and actuator dependencies
- downloaded zip
- unzipped in my downloads folder
- changed directory into the springboot project
- ran  `jx import`
- pressed enter when asked "? Would you like to initialise git now? Yes"
- pressed enter when asked "? Commit message:  Initial import"
 and I can see that the makefile has been added.

I wonder if there's a different step or project type?

**Comment by jstrachan** [comment]
what version of jx are you using? import is working in the BDD tests

**Comment by rawlingsj** [comment]
oops referenced the wrong issue in the fix above, re-opening

**Comment by cyrille-leclerc** [comment]
Problem fixed with jx-1.0.60
https://gist.github.com/cyrille-leclerc/e24072f45aa3d977a4123c996f2c10d7
